data/wanderers middle.txt

```diff
@@ -1766,7 +1766,73 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+				`	"How many ships were there? What did they look like?"`
+				`	"Where did this fight take place?"`
+					goto where
+				`	"You should be glad you're still in one piece."`
+					goto glad
+					
+			`	"Well," the pilot continues, "we had hired a beefy Leviathan escort to Farpoint in the Alnitak system. When we were going in for landing, two of these hostile ships jumped in from the northwest, the direction of all the anarchist colonies in this area. The ships weren't big, but they were tough——our four heavy laser turrets weren't enough to produce even a dent in their shields. They made short work of the Leviathan, and nearly disabled our ship, too. It was lucky I decided that jumping would be faster than landing. We warped out of the system while in atmosphere, saving the ship, but now I'm in debt to all my crew because I couldn't meet my rush delivery deadline."`
+			choice
+				`	"You're lucky as hell to be here today."`
+					goto glad
+				`	"Thanks for the information. I think that these hostile ships are the ones I'm looking for."`
+					goto thanks
+					
+			label where
+			`	"Well," the pilot explains, "we were doing a large cargo run to Farpoint in the Alnitak system, when two of these hostile ships jumped in from the northwest, the direction of all the anarchist colonies in this area. I had to jump the ship in atmosphere to stop us from getting shredded by their strange melting weaponry. I immediately reported the attack to a desk clerk at the Navy base in the neighboring system Saiph, but their servers show no logs of any unidentified ships beyond Farpoint, so I guess they turned around after scouting out the Alnitak system."`
+			choice
+				`	"You're lucky as hell to be here today."`
+					goto glad
+				`	"Thanks for the information. I think that these hostile ships are the ones I'm hunting for."`
+					goto thanks
+			
+			label thanks
+			`	"No problem, kiddo," the captain says, although he looks awed that you are even attempting to fight such powerful ships. "Remember, northwest of Farpoint. If they're anarchists of any sort, they'll be hiding in one of those pirate systems up there. Take care of yourself out there; those are some dangerous ships that jumped us." With that, he bids you farewell.`
+					decline
+					
+			label glad
+			`	"Yeah, those ships would have obliterated us if we hadn't jumped pronto." The pilot looks at you, then asks quizzically, "Say, do you know anything about those ships?"`
+			choice
+				`	"Yeah, those were Korath ships that attacked you."`
+				`	"Maybe. I have a feeling that they're the alien ships that I'm hunting."`
+				`	"I don't want to jump to any conclusions."`
+					goto no
+					
+			`	The pilot seems to display an expression of distaste upon hearing of aliens. You try to describe the dynamic between the exiles, the Efreti, Mereti, and Sestor, but to no avail whatsoever. The pilot quickly loses interest in your explanation, so you decide to steer the conversation to an end.`
```

Comment:
Perhaps he's disgusted that there are new aliens in human space! That would be a major point!

[My headcanon had the Captain be extremely experienced in the far north, so Hai. Had a scuffle with the Unfettered, hated aliens ever since. It's just a nuance that is up for the player to decide. I never hint at anything, so what this is trying to communicate is open to interpretation. Is he a xenophobe (literally)? Does he have bad memories of them? Is he disgusted that there are new aliens in human space?]